Makefile

```diff
@@ -75,3 +77,6 @@ www: \
 	dist/apple-touch-icon.png \
 	dist/index.html \
 	dist/manifest.webmanifest
+.PHONY: serve
```

Comment:
Here you should keep a space before the phony declaration. In general, see how we handle spaces in the rest of the Makefile and copy that approach